backend-shared/config/kcp/apiresourceschema_gitopsrvc.yaml

```diff
@@ -0,0 +1,436 @@
+# This file is generated from CRDs by ./hack/generate-kcp-api.sh script.
+# Please do not modify!
```

Comment:
Please include a Makefile target to regenerate these files when the CRD changes.